package.json

```diff
@@ -9,7 +9,7 @@
   },
   "main": "dist/npm/index.js",
   "module": "dist/es/index.js",
```

Comment:
@jensengar can you confirm that this is the file that worked in Web-dir? I think it was (and I am going to test it more just to confirm).

Reply:
i will, but it might not be until tomorrow.